app/Http/Middleware/Checks/HasAdminUser.php

```diff
@@ -19,16 +18,19 @@ class HasAdminUser implements MiddlewareCheck
 	 */
 	public function assert(): bool
 	{
-		try {
-			return User::query()->where('may_administrate', '=', true)->count() > 0;
-		} catch (QueryException $e) {
-			if (Str::contains($e->getMessage(), 'column "may_administrate" does not exist')) {
-				// We are doing an upgrade, therefore we can assume there exists an admin.
-				return true;
+		// This is necessary in the case that someone is using the web-updater.
+		// In such case the tables already exists so the IsInstaled will return true.
+		// However this middleware will throw an error because may_administrate does not exists yet.
```

Comment:
```suggestion
		// In such case the tables already exist so the IsInstalled will return true.
		// However this middleware will throw an error because may_administrate does not exist yet.
```